### DISCUSSION TITLE
SO CLOSE!!!

### DISCUSSION THREAD

**Discussion by Isaac-Smit** [Isaac-Smit/Conway-s-Game-of-Life-Group-1-#11] [General]
we were actually a lot closer than we thought we were. we just need to polish off some final things with the user input. e.g.. we need to properly do error handling such as things that aren't "yes/no" or a parameter that isn't between 0 - 10.

also we need to require the user to input a minimum of 3 seeds before the game loop starts, otherwise the world will die before as it starts!!!  @WilliamBus @777ParallelClematis

**Comment by 777ParallelClematis** [comment]
Sounds fantastic - should we randomise three input seeds to get the game started quicker for the user?

**Reply by Isaac-Smit** [reply]
we could do that. but i think it would be simpler to keep sequential and to make sure that 3 are required for the simulation to start.
